saffron/src/blob.rs

```diff
@@ -110,11 +114,51 @@ impl<G: KimchiCurve> FieldBlob<G> {
         bytes.truncate(blob.n_bytes);
         bytes
     }
+
+    #[instrument(skip_all, level = "debug")]
+    pub fn update<EFqSponge: Clone + FqSponge<G::BaseField, G, G::ScalarField>>(
+        &mut self,
+        srs: &SRS<G>,
+        domain: &Radix2EvaluationDomain<G::ScalarField>,
+        diff: Diff<G::ScalarField>,
+    ) {
+        let updates: Vec<(usize, PolyComm<G>, DensePolynomial<G::ScalarField>)> = diff
+            .evaluation_diffs
+            .into_par_iter()
+            .enumerate()
+            .map(|(index, d)| {
+                let d_p = {
+                    let evals = (0..domain.size())
+                        .map(|i| {
+                            d.get(&i)
+                                .copied()
+                                .unwrap_or(<G as AffineRepr>::ScalarField::zero())
```

Comment:
```suggestion
                                .unwrap_or(G::ScalarField::zero())
```
And you can remove `7 | use ark_ec::AffineRepr;`